### PR TITLE
Add rule for GitHub

### PR DESCRIPTION
**Pull request by TiagoDanin**
Related in #41 :+1:

### CODE REVIEW

File: rules/github.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+			headers,
+			json: true
+		});
+
+		const data = res.body;
+		if (!data.description) {
+			file.message('The repository should have a description');
+		}
+
+		if (!(data.topics.includes('awesome-list') && data.topics.includes('awesome'))) {
```
Comment (transitive-bullshit) [inline rules/github.js]: Let's make this two separate checks with separate invocations of `file.message`.

Comment (TiagoDanin) [reply rules/github.js]: @transitive-bullshit Done!

File: rules/github.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+		if (!data.description) {
+			file.message('The repository should have a description');
+		}
+
+		if (!data.topics.includes('awesome')) {
+			file.message('The repository should have "awesome" as GitHub topics');
+		}
+
+		if (!data.topics.includes('awesome-list')) {
+			file.message('The repository should have "awesome-list" as GitHub topics');
```
Comment (transitive-bullshit) [inline rules/github.js]: Let's change the English here to read:

`The repository should have "awesome-list" as a GitHub topic`

Same thing with the other message above

**Comment by transitive-bullshit** [comment]
I think this is good to merge after updating the message text.

@sindresorhus any thoughts on this or #75?

File: rules/github.js (hunk 2 above)
Comment (TiagoDanin) [reply rules/github.js]: Nice catch!

**Comment by transitive-bullshit** [comment]
Thanks again @TiagoDanin!

**Comment by sindresorhus** [comment]
Does this completely fix #41?

**Comment by transitive-bullshit** [comment]
I believe so.

**Pull request merged**